sdk/test-utils/recorder-v2/package.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "@azure/test-utils-recorder-v2",
+  "version": "1.0.0",
+  "sdk-type": "client",
+  "description": "This library provides interfaces and helper methods to provide recording and playback capabilities for the tests in Azure JS/TS SDKs",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "browser": {
+    "./dist-esm/src/basekarma.conf.js": "./dist-esm/src/basekarma.conf.browser.js",
+    "./dist-esm/src/createRecorder.js": "./dist-esm/src/createRecorder.browser.js",
+    "./dist-esm/src/utils/recordings.js": "./dist-esm/src/utils/recordings.browser.js"
+  },
+  "types": "./typings/src/index.d.ts",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "echo skipped",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "echo Skipped.",
+    "build:test": "tsc -p .",
+    "build": "tsc -p . && rollup -c 2>&1",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
+    "extract-api": "echo skipped",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "karma start --single-run",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace dist-esm/test/*.spec.js dist-esm/test/node/*.spec.js",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json src test --ext .ts -f html -o recorder-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/*.spec.ts\"",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "docs": "echo Skipped."
+  },
+  "files": [
```

Comment:
We never `npm pack` this package, so we could just remove this entry entirely. If not, s/typings/types.

Reply:
-> types.